spec/requests/authentications_spec.rb

```diff
@@ -1,5 +1,5 @@
 RSpec.describe 'Authentications API' do
-  let(:provider) { FactoryGirl.create(:provider_ansible_tower) }
+  let(:provider) { FactoryGirl.create(:automation_manager_ansible_tower) }
   let(:auth) { FactoryGirl.create(:ansible_cloud_credential, :resource => provider) }
   let(:auth_2) { FactoryGirl.create(:ansible_cloud_credential, :resource => provider) }
```

Comment:
👍 so if you fix the other parts in [here](https://github.com/jameswnl/manageiq-api/commit/35172b8dcf725287376f148113e48aeac0f0d3f6), other specs should pass as well.

Reply:
Thanks @kbrock !